packages/salesforcedx-vscode-core/package.json

```diff
@@ -305,6 +305,10 @@
         {
           "command": "sfdx.create.manifest",
           "when": "sfdx:project_opened"
+        },
+        {
+          "command": "sfdx.lightning.rename",
+          "when": "sfdx:project_opened && resource =~ /.*/(lwc|aura)/[^/]+(/[^/]+\\.(html|css|js|xml|svg|cmp|app|design|auradoc))?$/"
```

Comment:
If I"m reading this correctly, the command is available when the parent is named lwc or aura, and the file type is html, css, js, xml, svg, cmp, app, design, auradoc.  Wouldn't we want to include all files?  What if the file type was .TS, or .PNG?  Are there any file types we'd want to exclude?  If we want to include all, can the RegEx be changed to include all?  Maybe this:
```
"when": "sfdx:project_opened && resource =~ /.*/(lwc|aura)/[^/]+(/[^/]+\\.*)?$/"
```

RegEx isn't my strongest suit, but I thin you will get the idea.

Reply:
From my understanding, if users need JPG files, they have to store them in somewhere else like the assets folder that is located outside of lwc and aura component folder.

Reply:
OIC, Yea, images are normally in the `staticresources` directory, and I hadn't heard about images being within the components directory.  I looked into it, and the image is used for an icon in AppBuilder.